pcap-int.h

```diff
@@ -107,6 +107,107 @@ extern "C" {
  */
 #define MAXIMUM_SNAPLEN		262144
 
+/*
```

Comment:
Libpcap no longer has a `pcap_md` structure; instead, a `pcap_t` has a `void *` pointer to a private data structure, and each `pcap-XXX.c` file has its own private data structure, defined internally in the `pcap-XXX.c` file, to which that points.  We will not reintroduce the `pcap_md` structure.